README.md

```diff
@@ -82,7 +91,9 @@ AWS_SESSION_TOKEN=# passed to Transmogrifier containers for use
 
 ```text
 WEBAPP_HOST=# host for flask webapp
-WEBAPP_PORT# port for flask webapp
+WEBAPP_PORT=# port for flask webapp
+TRANSMOGRIFIER_MAX_WORKERS=# max number of Transmogrifier containers to run in parallel; default is 6
```

Comment:
"in parallel" -> "concurrently" / "asynchronously"?

Reply:
Technically, they _are_ running in parallel!

Though we orchestrate their starting / stopping / reporting via **concurrent** threads, the docker containers are detached and running as distinct OS-level processes in **parallel**.  

As such, I think "parallel" here is accurate, as a developer would need to know that setting this to "10" as an example, suggests you truly have 10 docker containers running in parallel, at the same time, gobbling up system resources.

Reply:
Following up here a bit, as other comments also touched on this.

From a high level, I think that env vars and doc strings should convey that these things are running in **parallel** because that's the ultimate strain on the system.

Technically, we are coordinating their starting + stopping via threads, which is concurrent, but that's kind of an implementation detail in the overall scheme of things.

Reply:
That makes sense. Thank you for this perspective!